db/migrate/20170814023144_add_volume_reference_on_chapters.rb

```diff
@@ -0,0 +1,6 @@
+class AddVolumeReferenceOnChapters < ActiveRecord::Migration
+  def change
+    add_reference :chapters, :volume, index: true,
+                                      foreign_key: true, null: true, default: nil, required: true
```

Comment:
Is there a reason for `null: true, default: nil, required: true`?

I'm pretty sure that's the default value for `null` and `default` and then `required` isn't even an option on this